models/ssh_key.go

```diff
@@ -740,6 +736,12 @@ func GetDeployKeyByRepo(keyID, repoID int64) (*DeployKey, error) {
 	return key, nil
 }
 
+// UpdateDeployKeyCols updates deploy key information only columns.
```

Comment:
nit: `// UpdateDeployKeyCols updates deploy key information only in the specified columns`

Reply:
done.